ZenPacks/zenoss/OpenStackInfrastructure/utils.py

```diff
@@ -313,4 +313,14 @@ def getNetSubnetsGws_from_GwInfo(external_gateway_info):
 
     return (network, subnets, gateways)
 
+def isValidHostname(hostname):
```

Comment:
Jake Lampe had a customer with bogus hostname (something like xxx.yyy.com:xxxxx), and he kept asking Phil and me about why modeler would not work. So I thought we would verify for decent hostname